format/spec.md

```diff
@@ -337,8 +337,11 @@ The set of metadata columns is:
 | **`2147483546  file_path`**      | `string`      | Path of a file, used in position-based delete files                                                    |
 | **`2147483545  pos`**            | `long`        | Ordinal position of a row, used in position-based delete files                                         |
 | **`2147483544  row`**            | `struct<...>` | Deleted row values, used in position-based delete files                                                |
-| **`2147483543  _row_id`**        | `long`        | A unique long assigned when row-lineage is enabled, see [Row Lineage](#row-lineage)                    |
-| **`2147483542  _last_updated_sequence_number`**   | `long`        | The sequence number which last updated this row when row-lineage is enabled [Row Lineage](#row-lineage) |
+| **`2147483543  _change_type`**                    | `string`      | Used in Changelog: The change type for this particular row:  INSERT, DELETE, UPDATE_BEFORE, or UPDATE_AFTER |
```

Comment:
I'd probably structure these statements as:
- `The record type in changelog (INSERT, ...)`
- `The snapshot ID in which the change occurred`

Reply:
+1 for the rewording and removal of `Used in changelog` to be consistent with other descriptions in the same table.

Reply:
Fixed